autosubmit/job/job.py

```diff
@@ -1059,15 +1059,16 @@ def _get_from_total_stats(self, index):
         :return: list of values in column index position
         :rtype: list[datetime.datetime]
         """
-        log_name = os.path.join(self._tmp_path, self.name + '_TOTAL_STATS')
+        log_name = Path(f"{self._tmp_path}/{self.name}_TOTAL_STATS")
         lst = []
-        if os.path.exists(log_name):
+        if log_name.exists() and log_name.stat().st_size > 0:
             f = open(log_name)
             lines = f.readlines()
             for line in lines:
                 fields = line.split()
                 if len(fields) >= index + 1:
                     lst.append(parse_date(fields[index]))
+            f.close()
```

Comment:
Probably simpler/better to use a context manager, https://docs.python.org/3/reference/compound_stmts.html#with, like `with open ... as f:`

This way we don't have to worry about closing it... as there could be an error before this new `f.close()` is called too…